### ISSUE TITLE
Enhance ics export

### ISSUE THREAD

**Issue by amberridge** [YACS-RCOS/yacs#464]
**Is your feature request related to a problem? Please describe.**
Importing the ics file causes the class sessions to go on weekly forever.
 
**Describe the solution you'd like**
Since YACS knows the last day of classes, add the UNTIL clause to the RRULE.  For example, this would end on December 12th at midnight GMT.

RRULE:FREQ=WEEKLY;UNTIL=20191212T000000Z

**Describe alternatives you've considered**
Would be nice if it also started on the first day of classes, but that is less of an issue overall.

**Additional context**
None.

**Comment by Malorn44** [comment]
I assume you were using yacs.cs.rpi.edu?

This site is being deprecated soon in favor of rpi.yacs.io

This new site doesn't have ics files implemented yet but we will try to have this feature there for when it gets implemented.

**Comment by amberridge** [comment]
Yes.  I didn't know about rpi.yacs.io.  Looks nice.  Slightly off topic, what is the "Topics" link supposed to do?
